src/controllers/categoryOfMeasuring.js

```diff
@@ -0,0 +1,11 @@
+const createCategoriesOfMeasuringsModel = require("../models/categoriesofmeasurng.js");
+
+const { DataTypes } = require("sequelize");
+const { db } = require("../databaseSetup");
+
+const CategoriesOfMeasurings = createCategoriesOfMeasuringsModel(db, DataTypes);
+
+const getAllCategoriesOfMeasurings = async (req, res) =>
+  res.json(await CategoriesOfMeasurings.findAll());
```

Comment:
Separate this code line into multiple ones for more readability.

Reply:
Separated ✔️